### DISCUSSION TITLE
Bundle without Cache

### DISCUSSION THREAD

**Discussion by Coder2195** [parcel-bundler/parcel#6540] [Q&A] [answered]
How do I use parcel to bundle without cache?
I’m using replit due to limitations of iOS
Despite having hacker status, having 200mb worth of cache isn’t my idea of fun
How do I bundle without cache?

**Comment by devongovett** [comment, marked as answer]
It's not really possible. --no-cache disables reading from the cache but not writing to it. We use the file system as a sort of temporary data store between different parts of a build to avoid allocating lots of memory. You can safely delete the .parcel-cache folder after a build to free up space though.

**Reply by Coder2195** [reply]
Could you have a fix for it?

**Reply by Coder2195** [reply]
And how do I delete it after if I’m using the command line?
Is there a way to delete without running two shells

**Reply by msimpson** [reply]
@Coder2195Text

You could use `--cache-dir /tmp/<repo-name>/.parcel-cache` so it's out of mind.

On MacOS, if I remember correctly, unused items are cleared every few days.

On *nix, `/tmp` is likely a RAMDisk and therefore cleared every shutdown/reboot.

**Reply by Coder2195** [reply]
I’m using replit tho

**Reply by msimpson** [reply]
Replit has a `/tmp` directory.

[You can see it in their documentation for playing audio.](https://docs.replit.com/misc/playing-audio-replit#:~:text=/tmp)

[And Node can reach it.](https://replit.com/@melvin2016/temporary-files-directory-in-nodejs)
